tests/js/spec/components/events/eventCause.spec.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import {Client} from 'app/api';
+import EventCause from 'app/components/events/eventCause';
+
+jest.mock('app/api');
+
+describe('EventCause', function() {
+  let wrapper, organization, project, event;
+
+  afterEach(function() {
+    Client.clearMockResponses();
+  });
+
+  beforeAll(function() {
+    event = TestStubs.Event();
+    organization = TestStubs.Organization();
+    project = TestStubs.Project();
+    Client.addMockResponse({
+      method: 'GET',
+      url: `/projects/${organization.slug}/${project.slug}/events/${event.id}/committers/`,
+      body: {
+        committers: {
+          commits: [
+            {
+              message:
+                'feat: Enhance suggested commits and add to alerts\n\n- Refactor components to use new shared CommitRow\n- Add Suspect Commits to alert emails\n- Refactor committers scanning code to handle various edge cases.',
+              score: 4,
+              id: 'ab2709293d0c9000829084ac7b1c9221fb18437c',
+              repository: TestStubs.Repository(),
+              dateCreated: '2018-03-02T18:30:26Z',
+            },
+          ],
+        },
+      },
+    });
+
+    wrapper = mount(
+      <EventCause event={event} orgId={organization.slug} projectId={project.slug} />,
+      {
+        context: {
+          organization,
+          project,
+          group: TestStubs.Group(),
+        },
+      }
+    );
+  });
+
+  it('renders', function(done) {
+    wrapper.update();
+
+    setTimeout(() => {
```

Comment:
did it not work without the timeout?

Reply:
nope -- i dont know what a better solution is but i had used this pattern in zeus

its kind of janky tbqh

Reply:
the solution might be using shallow instead of mount?

Reply:
nvm